subprojects/docs/src/docs/userguide/ideaPlugin.adoc

```diff
@@ -58,97 +56,66 @@ The IDEA plugin also adds a `cleanIdea` task to the project. This task deletes t
 The IDEA plugin adds the tasks shown below to a project. Notice that the `clean` task does not depend on the `cleanIdeaWorkspace` task. This is because the workspace typically contains a lot of user specific temporary data and it is not desirable to manipulate it outside IDEA.
 
 [[ideatasks]]
-.IDEA plugin - Tasks
-[cols="a,a,a,a", options="header"]
-|===
-| Task name
-| Depends on
-| Type
-| Description
-
-| `idea`
-| `ideaProject`, `ideaModule`, `ideaWorkspace`
-| -
-| Generates all IDEA configuration files
-
-| `openIdea`
-| `idea`
-| -
-| Generates all IDEA configuration files and opens the project in IDEA
-
-| `cleanIdea`
-| `cleanIdeaProject`, `cleanIdeaModule`
-| api:org.gradle.api.tasks.Delete[]
-| Removes all IDEA configuration files
-
-| `cleanIdeaProject`
-| -
-| api:org.gradle.api.tasks.Delete[]
-| Removes the IDEA project file
-
-| `cleanIdeaModule`
-| -
-| api:org.gradle.api.tasks.Delete[]
-| Removes the IDEA module file
-
-| `cleanIdeaWorkspace`
-| -
-| api:org.gradle.api.tasks.Delete[]
-| Removes the IDEA workspace file
-
-| `ideaProject`
-| -
-| api:org.gradle.plugins.ide.idea.GenerateIdeaProject[]
-| Generates the `.ipr` file. This task is only added to the root project.
-
-| `ideaModule`
-| -
-| api:org.gradle.plugins.ide.idea.GenerateIdeaModule[]
-| Generates the `.iml` file
-
-| `ideaWorkspace`
-| -
-| api:org.gradle.plugins.ide.idea.GenerateIdeaWorkspace[]
-| Generates the `.iws` file. This task is only added to the root project.
-|===
+`idea` — depends on: `ideaProject`, `ideaModule`, `ideaWorkspace`::
+Generates all IDEA configuration files
+
+`openIdea` — depends on: `idea`::
+Generates all IDEA configuration files and opens the project in IDEA
+
+`cleanIdea` — type: api:org.gradle.api.tasks.Delete[], depends on: `cleanIdeaProject`, `cleanIdeaModule`::
```

Comment:
similarly this formatting here is a little different than what we're using for the Java Plugin. I slightly prefer the format of the Java Plugin tasks because it's highly likely the header will fit on one line, but I see the merit in having depends on declared tasks here too.

Reply:
I think the 'depends on' information is relevant and important and should probably be added to the Java Plugin information. I don't know whether it's best to put this information on the same line as the task name. Might be able to give it its own line afterwards, independent of the description paragraph. I might experiment.

BTW, what do you think of using 'type: none' rather than 'type: Task'? All tasks are tasks after all, and one never specifies `type: Task` in a build script.

Reply:
![screenshot_20180419-092943](https://user-images.githubusercontent.com/19075/38980315-72642f68-43b4-11e8-9248-21eddd58a8a6.png)

I don't like the wrap around with a long 'depends on' list when it's on the first line. On the other hand, I think the gap between the italicized depends on and the following paragraph is unnaturally large. It's preferable though.

Reply:
I actually prefer the look of the modified version, even on desktop. And the spacing issue is something that could potentially be fixed with a CSS change anyway. Let me try it out. I noticed that I tried something similar but slightly different with the Base Plugin chapter, so I should update that as well.

Reply:
Okay, go for it.